pkg/services/alerting/notifier.go

```diff
@@ -24,9 +24,38 @@ var newImageUploaderProvider = func() (imguploader.ImageUploader, error) {
 type NotifierPlugin struct {
 	Type            string          `json:"type"`
 	Name            string          `json:"name"`
+	Heading         string          `json:"heading"`
 	Description     string          `json:"description"`
+	Info            string          `json:"info"`
 	OptionsTemplate string          `json:"optionsTemplate"`
 	Factory         NotifierFactory `json:"-"`
+	Options         []Option        `json:"options"`
+}
+
+// Option holds information about options specific for the NotifierPlugin.
+type Option struct {
+	Element        string         `json:"element"`
+	InputType      string         `json:"inputType"`
```

Comment:
Not a blocker. But it would be Nice to make a constants and a type for things that have only a limited number of supported values.

There are .. too many ways to do this in go: https://blog.learngoprogramming.com/golang-const-type-enums-iota-bc4befd096d3 . 

But a fairly straight forward way would be to declare something like the following:

```
// InputType is the ... ? type for for an Options.
type InputType string

const (
	// InputTypeText will render a text field in the frontend, and should be used with ElementTypeInput ...
	InputTypeText = "text"
)
```

will give you documentation and autocomplete when coding with a go IDE, which makes it nice. Once can still do:

```
var InputTypeFoo = InputType("foo")
```

in a WIP PR without adding constants.

However, I would consider this extra credit.

Reply:
Yeah, I was thinking about this but never got around to figuring out how it's done in go